apps/client/components/ui/Dashboard/DashboardTop.tsx

```diff
@@ -0,0 +1,25 @@
+import Photo from '../photo'
+
+function DashboardTop({ firstName, lastName, title, photo }) {
+  return (
+    <div className="bg-neutral-100 justify-between items-center h-24 grid grid-cols-6">
+      <h1 className="text-3xl text-green-mediumgreen font-extrabold col-span-4">
+        Team member Panel
```

Comment:
this should be dynamic too, no? so we could use the Dashboard Top for the User Dashboard and for the Manager Dashboard

Reply:
thats the next step